src/Layout/Layout.js

```diff
@@ -240,7 +240,7 @@ Layout.propTypes = {
    * Defines the `align-items` style property.
    * It's applied for all the screen sizes.
    */
-  xsAlign: PropTypes.oneOf([
+  align: PropTypes.oneOf([ // eslint-disable-line react/sort-prop-types
```

Comment:
I was just keeping them grouped together, otherwise `wrap` will be between `lg` and `xl`